pymod/ams.py

```diff
@@ -269,7 +280,7 @@ def _retry_make_request(self, url, body=None, route_name=None, retry=0,
                 finally:
                     i += 1
 
-    def _make_request(self, url, body=None, route_name=None, **reqkwargs):
+    def _make_request(self, url, body=None, route_name=None, params=None, **reqkwargs):
```

Comment:
I would say that introducing another argument here `params=None` is redundant and not needed as `reqkwargs` would "swallow" keyworded argument anyway and pass it to the underlying python-requests method. Same goes with outer method `_retry_make_request`